tests/features/java/java_s2i.feature

```diff
@@ -171,3 +171,11 @@ Feature: Openshift OpenJDK S2I tests
       Given s2i build https://github.com/jboss-container-images/openjdk from tests/OPENJDK-1549 with env using ubi9
        | variable           | value    |
```

Comment:
The environment table needs to be indented relative to the "Given" and "Then" lines, otherwise the "Then" line is not being interpreted correctly.